source/includes/markdown/_13_1-scim.html.md

```diff
@@ -31,10 +54,23 @@ The accepted user attributes are:
 |name.givenName|string|Unsupported, use `formatted`|
```

Comment:
Although it says "unsupported", it's only unsupported for modification - we still send these two props as part of the response... I think we should convey this distinction somehow.